scripts/index.js

```diff
@@ -0,0 +1,80 @@
+state = {
+    'temp': 72,
+}
+
+input.oninput = () => {
```

Comment:
I would add a function to reset the city name to the default city name. Right now no city appears on the page until I start typing one.